app/shared/types/settingsSchema.ts

```diff
@@ -119,6 +119,7 @@ export const settingsSchema = {
     customCSS: { type: 'string' },
     mapTilerKey: { type: 'string' },
     newNameGeneration: { type: 'boolean', enum: [true] },
+    toggleOCRButton: { type: 'boolean' },
```

Comment:
So, for sure, this name in the settings schema is wrong.  This is enabling the entire OCR service for a collection, not a button.